python/feathub/feathub_client.py

```diff
@@ -46,17 +46,17 @@ def __init__(self, props: Dict) -> None:
 
     def get_features(
         self,
-        features: Union[str, TableDescriptor],
+        feature_descriptor: Union[str, TableDescriptor],
```

Comment:
Can we follow the example provided in this thread [1] to rename the argument in a backward-compatible way?

[1] https://stackoverflow.com/questions/49802412/how-to-implement-deprecation-in-python-with-argument-alias